### PR TITLE
Get all animals

### PR DESCRIPTION
**Pull request by KMarienfeld**
first slice, getAllAnimals shows all Animals

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/controller/ControllerZooFoo.java [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("api")
+@RequiredArgsConstructor
+
```
Comment (SyntaxPanda) [inline backend/src/main/java/de/neuefische/backend/controller/ControllerZooFoo.java]: Pls delete the empty line, to much trigger for my self :'(

File: backend/src/main/java/de/neuefische/backend/repository/RepoAnimals.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+@Repository
+
+
+
+public class RepoAnimals {
+/*    Animals animals1 = new Animals("1", "Elephant", "fruit", 10, "Monday", 3, "open", "Amelie", "");
+    Animals animals2 = new Animals("2", "Giraffe", "fruit", 9, "Wednesday", 4, "open", "Svenja", "");
+    Animals animals3 = new Animals("3", "Bear", "fish", 10, "Friday", 1, "open", "Richard", "");
+    Animals animals4 = new Animals("4", "Monkey", "fruit", 5, "Thursday", 23, "open", "Kristina", "");*/
```
Comment (SyntaxPanda) [inline backend/src/main/java/de/neuefische/backend/repository/RepoAnimals.java]: Do you really need these lines in your code?

File: backend/src/main/java/de/neuefische/backend/repository/RepoAnimals.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    private final Map<String, Animals> allAnimalsMap = new HashMap<>( /*Map.of(
+            "1", animals1,
+            "2", animals2,
+            "3", animals3,
+            "4", animals4
+    )*/);
```
Comment (SyntaxPanda) [inline backend/src/main/java/de/neuefische/backend/repository/RepoAnimals.java]: Do you really need these lines in your code, too?

File: frontend/src/AnimalCard/AnimalCard.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import "./AnimalCard.css";
+import exp from "constants";
```
Comment (SyntaxPanda) [inline frontend/src/AnimalCard/AnimalCard.tsx]: What is exp? Do u really need this import?

File: frontend/src/AnimalCard/AnimalCard.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+export type Animal = {
+    id:string;
+    pictureOfAnimal:string;
+    species:string;
+    numberOfAnimals:number;
+    animalKeeper:string;
+    feedStatus:string;
+    food:string;
+    foodAmount:number;
+}
```
Comment (SyntaxPanda) [inline frontend/src/AnimalCard/AnimalCard.tsx]: Do u want outsource this type?

File: frontend/src/AnimalGallery/AnimalGallery.css
Comment (SyntaxPanda) [inline frontend/src/AnimalGallery/AnimalGallery.css:1]: Dominic meckert mich an das ich es melden soll -.-

Comment (KMarienfeld) [reply frontend/src/AnimalGallery/AnimalGallery.css:1]: kein Problem :D

File: frontend/src/AnimalCard/AnimalCard.tsx (hunk 5 above)
Comment (KMarienfeld) [reply frontend/src/AnimalCard/AnimalCard.tsx]: updatet

File: frontend/src/AnimalCard/AnimalCard.tsx (hunk 4 above)
Comment (KMarienfeld) [reply frontend/src/AnimalCard/AnimalCard.tsx]: wie didn't need it, thanks

**Review by SyntaxPanda — APPROVED** [review]
I like the most lines :)

**Review by Kreyno93 — APPROVED** [review]

**Comment by pmyshev** [comment]
class name in backend:
```
public class Animals {
    private String id;
    private String species;
    private String food;
    private int foodAmount;
    private String dayToFeed;
    private int numberOfAnimals;
    private String feedStatus;
    private String animalKeeper;
    private String pictureOfAnimal;

}
```

class name in frontend:
```
export type Animal = {
    id: string;
    pictureOfAnimal: string;
    species: string;
    numberOfAnimals: number;
    animalKeeper: string;
    feedStatus: string;
    food: string;
    foodAmount: number;
}
```

it would probably be a good idea to keep it homogeneous, and probably use the singular here better.

**Review by Kreyno93 — APPROVED** [review]

**Review by SvenjaKlenke — APPROVED** [review]

**Review by SyntaxPanda — APPROVED** [review]
Love this fast Support and changes :-)

**Pull request merged**